### PR TITLE
Add back github action to remove stale issues.

### PR DESCRIPTION
**Pull request by andygup**
Restores and updates the github action to remove stale issues. Updates include:
* Modified cron job to run at midnight UTC 
* Explicitly disables `delete-branch` for now
* Upgraded from `stale@v3` to `stale@v4`
* Updated all the issue messages to be in sync with the jsapi-feedback repo.

### CODE REVIEW

File: DotNet/.github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. If you need additional assistance please contact Esri Technical Support. Thank you for your contributions."
+          close-issue-message: "This issue has been automatically closed due to inactivity. If you need additional assistance please contact Esri Technical Support."
+          stale-pr-message: "This PR has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. If you need additional assistance please contact Esri Technical Support. Thank you for your contributions."
+          close-pr-message: "This PR has been automatically closed due to inactivity. If you need additional assistance please contact Esri Technical Support."          
+          only-labels: null
```
Comment (bsvensson) [inline DotNet/.github/workflows/stale.yml:24]: If the default value is "unset" for `only-labels` should we remove this entry line?

File: DotNet/.github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          days-before-stale: 1000
```
Comment (bsvensson) [inline DotNet/.github/workflows/stale.yml:25]: Should we maybe lower this to one year now?

**Review by bsvensson — APPROVED** [review]

File: DotNet/.github/workflows/stale.yml (hunk 1 above)
Comment (andygup) [reply DotNet/.github/workflows/stale.yml:24]: Yes, good catch!

File: DotNet/.github/workflows/stale.yml (hunk 2 above)
Comment (andygup) [reply DotNet/.github/workflows/stale.yml:25]: Sounds good, I'll lower it.

**Comment by andygup** [comment]
Updated. I also moved the yml file to the root dir, I'd accidentally placed it under /DotNet.

**Pull request merged**